eu.usdx.UltraStarDeluxe.yaml

```diff
@@ -0,0 +1,194 @@
+id: eu.usdx.UltraStarDeluxe
+branch: master
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+separate-locales: false
+appstream-compose: true
+rename-desktop-file: ultrastardx.desktop
+rename-icon: ultrastardx
+command: ultrastardx
+
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --socket=pulseaudio
+- --device=all
+- --filesystem=~/.java/.userPrefs/PortMidi:ro
+- --persist=.ultrastardx
```

Comment:
--share=ipc is missing. It's needed for x11 (or fallback)

Reply:
ok, although it works fine here without